docs/source/code_examples/tutorials/configurations/set_requests_settings.py

```diff
@@ -0,0 +1,7 @@
+from encord import EncordUserClient
+from encord.http.constants import RequestsSettings
+
+user_client: EncordUserClient = EncordUserClient.create_with_ssh_private_key(
+    "<your_private_key>",
+    requests_settings=RequestsSettings(max_retries=7, backoff_factor=0.3),
```

Comment:
Is `backoff_factor` something that everyone understands? We could consider hiding it as it has a default value (and a doc-string) which would help that one guy who really wants to use it.